pyproject.toml

```diff
@@ -23,7 +23,7 @@ dynamic = ["version", "readme"]
 dependencies = [
   "attrs",
   "morecantile>=3.1,<4.0",
-  "pygeos>=0.10",
+  "shapely>=2.0b2,<2.1",
```

Comment:
You could probably use `<3` if you trust their semver?

Reply:
well I don't trust mine 😂